LEGO1/lego/legoomni/include/legocarbuild.h

```diff
@@ -77,11 +74,19 @@ class LegoCarBuild : public LegoWorld {
 		return !strcmp(p_name, LegoCarBuild::ClassName()) || LegoWorld::IsA(p_name);
 	}
 
+	MxLong Notify(MxParam& p_param) override;         // vtable+0x04
```

Comment:
These should remain where they were before to maintain the ascending order of virtual function offsets

Reply:
oh, you're right, I missed that the other two are virtual functions as well. Will change that back.